general/press.rst

```diff
@@ -118,10 +118,38 @@ Blog Posts
   <http://www.cittametropolitanaroma.it/homepage/elenco-siti-tematici
   /wifimetropolitano/openwisp-la-soluzione-open-source-la-diffusione-
   servizi-wifi/>`_ (in Italian)
+
+Google Summer of Code Blog Posts
+--------------------------------
+
+2017 Students
+~~~~~~~~~~~~~
+
+- `Adding AirOS support to netjsonconfig <https://edoput.github.io/openwispgsoc/>`_ by Edoardo Putti.
+- `Building a Javascript Based Configuration UI for OpenWISP <https://medium.com/@gastonche/building-a-javascript-based-configuration-ui-for-openwisp-5eab15088a55>`_ by Nkhoh Gaston Che.
+- `OpenWISP 2 Network Topology <https://medium.com/@rohithasrk/openwisp-2-network-topology-gsoc-17-4765008ccba>`_ by Rohith A. S. R. K.
+- `Google Summer of Code 2017 Django-freeradius <https://delucafiorella2602.wordpress.com/>`_ by Fiorella De Luca.
+- `Raspbian backend for OpenWISP 2 <https://medium.com/@ritwickdsouza/gsoc-openwisp-raspbian-backend-for-openwisp-2-61ff91843362>`_ by Ritwick DSouza.
+
+
+2018 Students
+~~~~~~~~~~~~~
+
+- `OpenWISP IPAM: IP Address Management tool for OpenWISP2 <https://gist.github.com/anurag-ks/75d033c9652c559b065f9cc6320ea707>`_ by Anurag Sharma.
```

Comment:
Rohtith A.S.R.K's work missing from 2018, he didn't write a blog. :-(

Reply:
There's this one: https://medium.com/@rohithasrk/openwisp-2-network-topology-gsoc-17-4765008ccba

Another interesting gsoc project is here: https://medium.com/@ritwickdsouza/gsoc-openwisp-raspbian-backend-for-openwisp-2-61ff91843362

Reply:
> There's this one: https://medium.com/@rohithasrk/openwisp-2-network-topology-gsoc-17-4765008ccba

This one is added, it's for 2017, I couldn't find one for 2018.